style.css

```diff
@@ -92,18 +194,54 @@ footer {
         background-size: 50rem 30rem;
     }
 
-    .portfolioHighlightsContainer {
-        padding: 2rem 1rem;
+    .aboutMe {
+        font-size: 1.5rem;
+        margin: 0rem 3rem;
+    }
+
+    .portfolioBlurb {
+        font-size: 1.5rem;
+        margin: 0rem 3rem;
+    }
+}
+
+@media screen and (max-width: 500px) {
+
+    .jumbotron {
```

Comment:
fix this media query so that the content doesn't get cut off on the left side of the screen and overflow.